rpcs3/Emu/SysCalls/Modules/cellAvconfExt.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+namespace vm { using namespace ps3; }
+
+struct avconfext_t
+{
+	u8 gamma;
+
+	avconfext_t()
+	{
+		gamma = 1;
+	}
+};
```

Comment:
Please, do I need to point to every file without newline? It's not a big issue, but could you at least check other files as well if you fix it in one place?